server/src/util/__tests__/__snapshots__/sourcing.test.ts.snap

```diff
@@ -126,7 +126,7 @@ exports[`getSourcedUris returns a set of sourced files (but ignores some unhandl
         "line": 25,
       },
     },
-    "uri": "file:///Users/bash/issue206.sh",
+    "uri": null,
```

Comment:
What happened here. Cannot be parsed anymore?

Reply:
Sorry, marked this as draft hoping it wouldn't bother you until this was ready. That happened automatically when I updated the snapshots, I'm still poking to see what happened exactly

Reply:
No worries. And thanks for looking into this.

Reply:
Closing this convo since I undid this change, and marking this as ready for review. Thanks for your time!